syntax="proto3";
import "common.proto";

package wifi;

enum startstop_enum {
  START= 0;//Ok
  STOP = 1;//Not Ok  
}
  
message StartStop_req {
  startstop_enum startstop = 1;
  }
   
message StartStop_rep {
    ReturnCode retVal = 1 ; 
  }

message StartStop {
    oneof messages {
      StartStop_req req = 1;
      StartStop_rep rep = 2;
  }
}
message seq_req_rep {
  oneof messages {
    StartStop startstop_msg = 1;
  }
}
 